Basic-Car-Maintenance.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,9 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		ED1FE7692CB4BEDC006F861F /* FirebaseFirestoreSwift in Frameworks */ = {isa = PBXBuildFile; productRef = ED1FE7682CB4BEDC006F861F /* FirebaseFirestoreSwift */; };
```

Comment:
do we still need to import all the Firebase stuff into the widget? Or do we not need to now, and only use keychain?

Reply:
Yes, since the widget uses the models/enums to fetch data from Firestore. The keychain was needed to share the authentication between applications. [Firebase Cross-Application Authentication](https://firebase.google.com/docs/auth/ios/single-sign-on)

`FirebaseCore` is used to configure the firebase app.

`FirebaseAuth` is used to access the current user from `Auth.auth()`.

`FirebaseFirestoreSwift` is needed since we are including `Vehicle.swift` in the widget and that file uses a Firebase property wrapper.